### ISSUE TITLE
Verify that Theano and TF actually work with the GPU

### ISSUE THREAD

**Issue by ryanjulian** [rlworkgroup/garage#204]
I have my doubts.

**Comment by naeioi** [comment]
Since I have no mac I can only test on Linux. @pelillian Maybe could you help test on osx?

**Comment by ryanjulian** [comment]
GPU is irrelevant on Mac--we only support CPU there (no CUDA for Mac)

**Comment by naeioi** [comment]
I tested Tensorflow and Theano and they work properly with GPU on machine. 
However, extra package installation and configuration are needed to enable GPU for Theano. I am going to do a PR to address this issue. 

**Test environment**
OS: Ubuntu 18.04.1 x86_64 LTS
Kernel: 4.15.0-36-generic
Graphics Card: NVIDIA GeForce GTX 1050 (Notebook)
Driver: NVIDIA 396.54
CUDA: 9.0

Tested against all test suites in `tests/garage/tf` and `tests/garage/theano`.
